README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
-  <a href="https://github.com/leapfrogtechnology/chill">
-    <img width="128px" src="opensource-logo.png"/> 
+  <a href="">
+    <img width="128px" src="assests/opensource-logo.png"/> 
```

Comment:
O is it? `assests` 🙈 

```diff
- assests
+ assets
```

Reply:
I always get confused `assets` or `assests` :D